platforms/hermes-javascript/README.md

```diff
@@ -0,0 +1,654 @@
+
+#### feedback()
+
+Use the Sound Feedback Api Subset.
+
+```js
+const hermes = new Hermes()
+const feedback = hermes.feedback()
+```
+
+#### destroy()
+
+Release all the resources associated with this Hermes instance.
+
+```js
+const hermes = new Hermes()
+hermes.destroy()
+```
+
+### Common ApiSubset methods
+
+**Check out [the hermes protocol documentation](https://docs.snips.ai/ressources/hermes-protocol) for more details on the event names.**
+
+#### on(eventName, listener)
+
+Subscribes to an event on the bus.
+
+```js
+const hermes = new Hermes()
+const dialog = hermes.dialog()
+
+dialog.on('session_started', message => {
+    /* ... */
+})
+```
+
+#### once(eventName, listener)
+
+Subscribes to an event on the bus, then unsubscribes after the first event is received.
+
+```js
+const hermes = new Hermes()
+const dialog = hermes.dialog()
+
+dialog.once('intent/myIntent', message => {
+    /* ... */
+})
+```
+
+#### off(eventName, listener)
+
+Unsubscribe an already existing event.
+
+```js
+const hermes = new Hermes()
+const dialog = hermes.dialog()
+
+const handler = message => {
+    /* ... */
+}
+
+// Subscribes
+dialog.on('intent/myIntent', handler)
+
+// Unsubscribes
+dialog.off('intent/myIntent', handler)
+```
+
+#### publish(eventName, message)
+
+Publish an event programatically.
+
+```js
+const hermes = new Hermes()
+const dialog = hermes.dialog()
+
+dialog.publish('start_session', {
+    custom_data: 'some data',
+    site_id: 'site Id',
+    session_init: {
+        init_type: 1,
+        value: 'notification'
+    }
+})
+```
+
+### Dialog Api Subset
+
+The dialog manager.
+
+#### Events available for publishing
+
+- **start_session**
+
+Start a new dialog session.
+
+```js
+dialog.publish('start_session', {
+    custom_data: /* string */,
+    site_id: /* string */,
+    session_init: {
+        init_type: /* 1 or 2 */,
+        value:
+        /* If init_type is 1 */
+            /* string */
+        /* If init_type is 2 */
+            {
+                text: /* string */,
+                intent_filter: /* string[] */,
+                can_be_enqueued: /* boolean */,
+                send_intent_not_recognized: /* boolean */
+            }
+    }
+})
+```
+
+- **continue_session**
+
+Continue a dialog session.
+
+```js
+dialog.publish('continue_session', {
+    session_id: /* string */,
+    text: /* string */,
+    intent_filter: /* string[] */,
+    custom_data: /* string */,
+    send_intent_not_recognized: /* boolean */
+})
+```
+
+- **end_session**
+
+Finish a dialog session.
+
+```js
+dialog.publish('end_session', {
+    session_id: /* string */,
+    text: /* string */
+})
+```
+
+#### Events available for subscribing
+
+- **intent/[intentName]**
+
+An intent was recognized.
+
+- **session_ended**
+
+A dialog session has ended.
+
+- **session_queued**
+
+A dialog session has been put in the queue.
+
+- **session_started**
+
+A dialog session has started.
+
+- **intent_not_recognized**
+
+No intents were recognized.
```

Comment:
Add a note that this must be enabled for each dialogue turn in either start or continue session

Reply:
✅